File: webserver/webserver/changuito/changuito-war/src/main/java/ar/com/utn/changuito/model/ControlVueltoStatistic.java
package ar.com.utn.changuito.model;

import ar.com.utn.changuito.architecture.net.SharedObject;

public class ControlVueltoStatistic extends Statistic{

	public void setPago(final int pago){
		set("pago",pago);
	}
	public int getPago(){
		return getInt("pago");
	}

	public void setMonto(final int monto){
		set("montoFinal",monto);
	}
	
	public int getMonto(){
		return getInt("monto");
	}

	public void setFailedVuelto(final int failedVuelto){
		set("failedVuelto",failedVuelto);
	}
	
	public int getFailedVuelto(){
		return getInt("failedVuelto");
	}

	
	
	public ControlVueltoStatistic(){
		
	}
	
   public ControlVueltoStatistic(final SharedObject sharedObject) {
	        super(sharedObject);
   }
}

File: changuito/Assets/Scripts/Model/Statistics/ControlVueltoStatistic.cs
using UnityEngine;
using System.Collections;
using System;

public class ControlVueltoStatistic : Statistic {

    public int FailedVuelto
    {
        get { return GetInt("failedVuelto"); }
        set { Set("failedVuelto", value); }
    }

    public int Pago
    {
        get { return GetInt("pago"); }
        set { Set("pago", value); }
    }

    public int Monto
    {
        get { return GetInt("monto"); }
        set { Set("monto", value); }
    }

     public ControlVueltoStatistic(DateTime gameStart, int failedVuelto)
        : base(gameStart)
    {
        IdEvento = "fin_CV";
        IdPantalla = CV;
        FailedVuelto = failedVuelto;
        Pago = PagoStatus.pago;
        Monto = PagoStatus.monto;
     }

     public ControlVueltoStatistic()
     {

     }

}
